pandas/core/indexes/range.py

```diff
@@ -231,7 +231,7 @@ def _start(self):
             Use ``start`` instead.
         """
         warnings.warn(self._deprecation_message.format("_start", "start"),
-                      DeprecationWarning, stacklevel=2)
```

Comment:
I think the start/stop/step ones we should not yet change as those were only introduced in the upcoming release